Article/Article.js

```diff
@@ -14,7 +14,7 @@ class Article {
   }
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-    this.expandButton.classList.toggle('expandButton')
+    this.domElement.classList.toggle('article')
```

Comment:
Check the article.less file to see the exact name used to open the article because when you click expand it is busting out the container and losing the white background.